joml-geometry/src/main/java/org/terasology/joml/geom/Rectangled.java

```diff
@@ -170,6 +192,31 @@ public Rectangled set(Rectangled source){
         return this;
     }
 
+    /**
+     * Set this rectangle to be a clone of <code>source</code>.
+     *
+     * @param x the x coordinate of both minimum and maximum corner
+     * @param y the y coordinate of both minimum and maximum corner
+     * @return this
+     */
+    public Rectangled set(double x, double y) {
+        this.minX = x;
+        this.minY = y;
+        this.maxX = x;
+        this.maxY = y;
+        return this;
+    }
+
+    /**
+     * Set this rectangle to be a clone of <code>source</code>.
+     *
+     * @param point the coordinate of both minimum and maximum corner
```

Comment:
This should be in line with the other two. Or rather they with this respectively.